SonarQube.Client/SonarQubeService.cs

```diff
@@ -323,13 +326,18 @@ public async Task<SonarQubeQualityProfile> GetQualityProfileAsync(string project
                 },
                 token);
 
-        public async Task<IList<SonarQubeIssue>> GetTaintVulnerabilitiesAsync(string projectKey, CancellationToken token) =>
-            await InvokeRequestAsync<IGetTaintVulnerabilitiesRequest, SonarQubeIssue[]>(
+        public async Task<IList<SonarQubeIssue>> GetTaintVulnerabilitiesAsync(string projectKey, CancellationToken token)
+        {
+            var issues = await InvokeRequestAsync<IGetTaintVulnerabilitiesRequest, SonarQubeIssue[]>(
                 request =>
                 {
                     request.ProjectKey = projectKey;
                 }, token);
 
+            await secondaryIssueHashUpdater.UpdateHashesAsync(issues, this, token);
```

Comment:
This line is the only "real" change in the product code. Everything else is test-related plumbing.